src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
```

Comment:
I like how the methods are written neatly and is very easy to read and understand!